queue.tf

```diff
@@ -24,18 +24,28 @@ module "notification-queue" {
   resource_group_name = "${azurerm_resource_group.rg.name}"
 }
 
+# deprecated, use `envelopes_queue_primary_listen_connection_string` instead
```

Comment:
keeping old one to not introduce breaking changes.
will be removed once all apps transition to using the new output var.